### ISSUE TITLE
App not available anymore - any solution?

### ISSUE THREAD

**Issue by maxmar391** [jjrobots/B-ROBOT_EVO2#8]
I have dug out my jjrobot lately and wanted to play with it again. However, I cannot find the app in the App Store anymore. Any solution for this issue? Can I find the app somehow and install it on my phone?

This is very dissappointing, giving that the jjrobot kit wasn't really cheap and now after short time is does not seem to be usable anymore and generate unnecessary electronic garbage? If the app is not at all available anymore and the robot cannot be used anymore, the idea behind the robot is good, but the realization and business model are bad.

**Comment by John-Burbidge** [comment]
Yes, there are sources online to download then “side load” the app onto
your device.


J

**Comment by Belott** [comment]
Did you find the app? I still have the APK file and would be happy to provide it to you.

**Comment by maxmar391** [comment]
@John-Burbidge I found this one here: https://m.apkpure.com/jjrobots-control-app/jjrobots.jjrobots_002/download
But it seems to be for Android only. Forgive me, I am not very familiar with handling apk files. Do they only work on Android and what would be the equivalent on iOS (if even possible, I guess side loading is almost impossible on iOS).

@Belott can you publish the APK file somewhere? I am pretty sure there are even more people that are interested in the file. Or is it the same as the link above? I guess your file is also for Android, not for iOS, right?

**Comment by mar0x** [comment]
Google request for "jjrobots apk" gives https://apkpure.com/jjrobots-control-app/jjrobots.jjrobots_002 at the top.
